### ISSUE TITLE
Game Misinterpreting Joystick Axis Readings

### ISSUE THREAD

**Issue by Canis-Enigmas** [RenaKunisaki/CSL-JoystickCameraControl#5]
I have a Logitech G13 that has it's thumb joystick that I'm trying to use with this mod.
However, Unity Input Manager readout in the Devices tab is showing 
Horizontal: 80 to 90
Vertical: -80 to -100
when the joystick is left alone dead center.
As a result, my game is always flying off to the far bottom right of the map without me touching anything. Trying to get it to stay still would require me to position the joystick somewhere halfway between extreme top-left position.

I've tried repeatedly to calibrate it in Windows' Game Controller window, but it looks like it isn't a Windows or device problem as the calibration tests all show the joystick is still dead accurate and reliable and I still can use it outside the game without issue. 

I suspect that the game or Unity is thinking the device is reporting a single range per X & Y axis or something of that sort. Is this something wrong with Unity Input Manager or something I have to manually configure for it to recognize the device axis reporting correctly?

**Comment by RenaKunisaki** [comment]
You should be able to compensate by adding an offset, or by enabling Scan for USB Devices and using the joystick directly instead of through Unity.

**Comment by Canis-Enigmas** [comment]
No, neither of these options do anything. Scan for USB Devices pops up a additional X & Y Axis readout in Devices, but the joystick readouts are fluctuating in the Unity Input Manger's Horizontal and Vertical fields. The Offset option doesn't do anything to them at all. 

Even worse still, whenever I switch anything in the Inputs tab, the result is clicking my mouse button will position it's cursor dead center on the screen and send the click event there. Rendering my only control in the game completely useless, leaving me with Ctrl-Alt-Del as the only way to do anything and exit the game.

**Comment by RenaKunisaki** [comment]
I think you should still be able to quit the game using Alt+F4?

Can you take a screenshot of each of the settings tabs so I can understand
exactly how you have it set up?

**Comment by Canis-Enigmas** [comment]
This is what my initial settings are:
![image](https://user-images.githubusercontent.com/14925968/82309000-e4a8c080-9a05-11ea-8c3d-e7809d1800e9.png)
![image](https://user-images.githubusercontent.com/14925968/82310066-4158ab00-9a07-11ea-81bb-9c8907485227.png)

Note the joystick is left dead center and these are the readouts:
![image](https://user-images.githubusercontent.com/14925968/82309345-5bde5480-9a06-11ea-8f28-44fd568b94db.png)
I've set the Offset settings for the Horizontal and Vertical inputs to the extreme settings
![image](https://user-images.githubusercontent.com/14925968/82309442-7adce680-9a06-11ea-9411-c22fad48792e.png)
But nothing changes
![image](https://user-images.githubusercontent.com/14925968/82309621-b081cf80-9a06-11ea-985d-6d52e4e6efee.png)
The mouse locking in the center seems to only occur in game. It doesn't occur in the Main Menu.

**Comment by RenaKunisaki** [comment]
I see. This might be an issue with Unity itself. Those inputs come from the game, and I don't think it's possible to override them. There might be settings within Steam to stop Unity from using the joystick; it should still work with the mod.

**Comment by Canis-Enigmas** [comment]
Damn. Are there any guides or tools you could suggest or point me to search for? 
I don't have any knowledge of Unity's design or development.
